projects/plugins/jetpack/modules/stats.php

```diff
@@ -588,6 +717,184 @@ function stats_reports_page( $main_chart_only = false ) {
 	}
 }
 
+/**
+ * Legacy Stats: Print Header Section
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
+function stats_print_header_section( $html ) {
+	$header = stats_parse_header_section( $html );
+	echo $header; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+}
+
+/**
+ * Legacy Stats: Print Content Section
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
+function stats_print_content_section( $html ) {
+	$content = stats_parse_content_section( $html );
+	echo $content; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+}
+
+/**
+ * Legacy Stats: Print Chart Scripts
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
+function stats_print_chart_scripts( $html ) {
+	$b = stats_content_marker();
+	$i = strpos( $html, $b );
```

Comment:
Could we please use a more descriptive name here? For example, `is_chart_scripts` if I didn't get it wrong. A few other places would need some renaming too.

Reply:
In this case, it actually prints the JavaScript into the output if it's found. It's checking the content first and either returning early (no output) or it finds the scripts it dumps them into the output stream.

Reply:
ya that’s exactly the reason we’d need a more descriptive name eh?